samples/client/petstore/kotlin-allOff-discriminator/src/main/kotlin/org/openapitools/client/models/Bird.kt

```diff
@@ -0,0 +1,44 @@
+/**
+ * Example
+ *
+ * An example
+ *
+ * The version of the OpenAPI document: 0.1
+ * Contact: contact@example.org
+ *
+ * Please note:
+ * This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * Do not edit this file manually.
+ */
+
+@file:Suppress(
+    "ArrayInDataClass",
+    "EnumEntryName",
+    "RemoveRedundantQualifierName",
+    "UnusedImport"
+)
+
+package org.openapitools.client.models
+
+import org.openapitools.client.models.Animal
+import org.openapitools.client.models.BirdAllOf
+
+import com.squareup.moshi.Json
+
+/**
+ * 
+ *
+ * @param id 
+ * @param featherType 
+ */
+
+data class Bird (
+
+    @Json(name = "id")
+    override val id: java.util.UUID,
+
+    @Json(name = "featherType")
+    val featherType: kotlin.String
+
+) : Animal
```

Comment:
Without this fix, this produced `Animal()` whish fail because `Animal` does not hava a constructor.